package.json

```diff
@@ -29,8 +29,9 @@
     "build": "tsc",
     "build:clean": "shx rm -rf ./dist ./coverage ./.nyc_output",
     "lint": "tslint --project .",
+    "test-lint": "tslint \"src/**/*.spec.ts\" && tslint \"src/test-helpers.ts\"",
```

Comment:
I noticed tslint has not been enabled for unit tests. I've enabled it and fixed all the warnings.

Reply:
@stevengill In my understanding, the reason why they're excluded is to avoid including spec files in npm package. Do you have some ideas to simplify the configuration without breaking anything?

Reply:
Oh I didn't know that. Thanks for the explanation.